$MirOS: ports/devel/cvs/patches/patch-MINOR-BUGS,v 1.2 2010/09/19 19:59:30 tg Exp $
--- MINOR-BUGS.orig	Tue Dec  2 22:14:45 1997
+++ MINOR-BUGS	Thu Jul 28 15:45:48 2011
@@ -59,3 +59,17 @@ particular reason to think the bug is fi
   belong under the "checkout" function?  Perhaps it is more logically
   grouped with the "history" function or we should create a new "info"
   function?
+
+
+Note that it is the opinion of the MirBSD founder that the CVSROOT
+environment variable ought to *never* be set at all. This prevents
+a lot of trouble. Almost all CVS tutorials are perceived bogus.
+
+This version appears to have trouble with the testsuite for write
+proxies. Other than that, everything appears to work well.
+
+"cvs annotate -b" does not do what people would expect, I think,
+but neither does it do that under MidnightBSD/DragonFly, where
+it came from.
+
+$MirOS: ports/devel/cvs/patches/patch-MINOR-BUGS,v 1.2 2010/09/19 19:59:30 tg Exp $
